doc/operations/troubleshooting.rst

```diff
@@ -0,0 +1,81 @@
+Troubleshooting CCF
+===================
+
+This page contains troubleshooting tips for CCF.
+
+Tips for interacting with CCF to diagnose issues
+------------------------------------------------
+.. note:: In the examples below this documentation uses ``example-ccf-domain.com`` as an example CCF domain, you will need to replace that with your own CCF domain when using these commands. 
+
+.. note:: CCF may be deployed with a loadbalancer which may cache the node which last responded to a query from an IP address. Until the cache clears, any subsequent queries from that IP address will be handled by the same node. As an example, if the cache clears after one minute, then in order to get a response from a different node, an operator must wait one minute between queries.  
+
+Below are descriptions of CLI commands and how they are useful for diagnosing CCF issues:
+
+**“What node is handling my requests?”**
+
+.. code-block:: bash 
+
+    curl -k https://example-ccf-domain.com/node/network/nodes/self -i
```

Comment:
We usually discourage users/members/operators to use `curl -k`, which means that the service identity is not verified, although we use it in practice so that we don't have to find the service certificate.

As you can see [here](https://microsoft.github.io/CCF/main/use_apps/issue_commands.html#issuing-commands), we recommend users to specify the `--cacert` option, which points to the service certificate.

Reply:
+1 for avoiding -k as much as possible in the documentation, particularly outside a strict operation context with local IP addresses.

Reply:
Thanks I've removed the `-k` and informed readers, in the note at the top of the page, they will need to add authentication parameters to the curl commands, as in the example [here](https://microsoft.github.io/CCF/main/use_apps/issue_commands.html#issuing-commands)